cmd/manager/main.go

```diff
@@ -114,23 +114,25 @@ func main() {
 
 	registryAPI := registry.NewRegistry()
 	jobHelperAPI := utils.NewJobHelper(client)
+	buildHelperAPI := build.NewHelper()
+	signHelperAPI := sign.NewSignerHelper()
```

Comment:
That variable is not strictly needed as it's used only once.

Reply:
fixed